README.md

```diff
@@ -335,6 +336,9 @@ Of course, fosite ships handlers for all OAuth2 and OpenID Connect flows.
  [Authentication using the Implicit Flow](http://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowAuth),
  [Authentication using the Hybrid Flow](http://openid.net/specs/openid-connect-core-1_0.html#HybridFlowAuth)
  
+### JWT Introspection
+
+Please note that when using the OAuth2StatelessJWTIntrospectionFactory access token revocation is not possible.
 
 This section is missing documentation and we welcome any contributions in that direction.
```

Comment:
You mean this line: "This section is missing documentation and we welcome any contributions in that direction." ? I thought it was a line belonging to the documentation chapter in general. If it was meant for the "Extensible handlers" then i'll move it above the "JWT Introspection".